internal/pkg/manifest/worker_svc.go

```diff
@@ -207,9 +238,22 @@ func (s *WorkerService) EnvFile() string {
 // Subscriptions returns a list of TopicSubscriotion objects which represent the SNS topics the service
 // receives messages from.
 func (s *WorkerService) Subscriptions() []TopicSubscription {
+	s.RetrofitFIFOConfig()
 	return s.Subscribe.Topics
 }
 
+// RetrofitFIFOConfig appends ".fifo" suffix to fifo topics.
+func (s *WorkerService) RetrofitFIFOConfig() {
```

Comment:
We probably no longer need to export this method now.

Also, I think we should be cautious about changing the value of `s.Subscribe.Topics[idx].Name` here. That is a user input, I feel like we should keep the user input as is. Besides, changing the value seems like a "side effect" of the `Subscriptions()` method. It is not obvious to the callers that calling `Subscription` would cause changes of the underlying data within `s *WorkerService`.

What do you think of:
```go
subs := make([]TopicSubscription, len(s.Subscribe.Topics))
for idx, topic := range s.Subscribe.Topics {
                 var topicSubscription TopicSubscription
                 // name = aws.StringValue(topic.Name) 
	         // if fifo { name = aws.String(aws.StringValue(topic.Name) + ".fifo") }
                 // topicSubscription.Name = name
                 subs = append(subs, topicSubscription)
}
return subs

```

Reply:
Sounds good to me